.github/workflows/main.yml

```diff
@@ -9,9 +9,6 @@ on:
 
 jobs:
   ci:
-    strategy:
-      fail-fast: true
-
```

Comment:
I made a mistake, this is meaningless since we don't have any matrix here. Removing it along with the CI fix.